packages/integration-tests/src/tests/api/sign-in-experience.test.ts

```diff
@@ -34,6 +34,7 @@ describe('admin console sign-in experience', () => {
       mfa: {
         policy: MfaPolicy.PromptAtSignInAndSignUp,
         factors: [],
+        organizationRequiredMfaPolicy: OrganizationRequiredMfaPolicy.Mandatory,
```

Comment:
Can we make `policy` to receive an array of policies, and add organization related mfa policy as a new enumeration item? E.g.:
```ts
policy: [MfaPolicy.PromptAtSignInAndSignUp, MfaPolicy.RequiredByOrganization]
```

Reply:
These two settings serve different purposes. Additionally, organizationRequiredMfa may apply different prompt policies, such as prompting during sign-in with the option to skip.